libnavigation-core/src/main/java/com/mapbox/navigation/core/MapboxNavigation.kt

```diff
@@ -925,16 +937,29 @@ class MapboxNavigation(
         @FeedbackEvent.Source feedbackSource: String,
         screenshot: String?,
         feedbackSubType: Array<@FeedbackEvent.SubType String>? = emptyArray(),
+        feedbackMetadata: FeedbackMetadata? = null,
     ) {
         MapboxNavigationTelemetry.postUserFeedback(
             feedbackType,
             description,
             feedbackSource,
             screenshot,
             feedbackSubType,
+            feedbackMetadata,
         )
     }
 
+    /**
+     * Provides wrapper of [FeedbackMetadata]. [FeedbackMetadata] is used to send deferred
+     * feedback attached to passed location. It contains data(like location, time) when method is
+     * invoked.
+     *
+     * Note: method throws [IllegalStateException] if trips session is not
+     * started ([startTripSession])
+     */
+    fun provideFeedbackMetadataWrapper(): FeedbackMetadataWrapper =
```

Comment:
Should we mark this as experimental and test in 1TAp first that the APIs and behavior are working as we expect?

Reply:
Also, we might want to verify how these APIs integrate into the whole upcoming Feedack UI system. That's why I feel like it might be safer to mark the metadata, metadata wrapper, and getter as experimental for now, until the Feedback UI is introduced. What do you think?

Reply:
nice suggestion 👍 also added separate method `postUserFeedback` with experimental annotation and parameter `FeedbackMetadata`